gamestonk_terminal/decorators.py

```diff
@@ -9,13 +9,7 @@
 
 
 def try_except(f):
-    """Adds a try except block if the user is not in development mode
-
-    Parameters
-    -------
-    f: function
-        The function to be wrapped
-    """
+    """Adds a try except block if the user is not in development mode"""
     # pylint: disable=inconsistent-return-statements
```

Comment:
Lol. I was actually talking about the `#pylint: ` thing. The return statements are consistent in this file (at least my pylint doesn't throw errors on my end

Reply:
I can add function back. Sometimes it returns a function and sometimes it returns an empty list